server/http_status.go

```diff
@@ -280,7 +280,7 @@ func (s *Server) setupStatuServerAndRPCServer(addr string, serverMux *http.Serve
 	grpcL := m.Match(cmux.Any())
 
 	s.statusServer = &http.Server{Addr: addr, Handler: CorsHandler{handler: serverMux, cfg: s.cfg}}
-	s.grpcServer = NewRPCServer(s.cfg.Security)
+	s.grpcServer = NewRPCServer(s.cfg.Security, s.dom, s)
```

Comment:
Seems the signature can be simplified to `NewRPCServer(s)`

Reply:
no, the signature is hard use in test `setUpRPCService`.